app/views/layouts/_navbar.html.haml

```diff
@@ -0,0 +1,28 @@
+%nav.navbar.navbar-default
+  .container-fluid
+    / Brand and toggle get grouped for better mobile display
+    .navbar-header
+      %button.navbar-toggle.collapsed{"aria-expanded" => "false", "data-target" => "#bs-example-navbar-collapse-1", "data-toggle" => "collapse", :type => "button"}
```

Comment:
Line is too long. [163/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace